src/securejoin.rs

```diff
@@ -1396,4 +1398,50 @@ First thread."#;
         let contact_alice = Contact::get_by_id(&bob, contact_alice_id).await.unwrap();
         assert_eq!(contact_alice.is_verified(&bob).await.unwrap(), true);
     }
+
+    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
```

Comment:
I think that this test should get a comment, along the lines of:

```rust
    /// An unencrypted message with already known Autocrypt key, but sent from another address, means that
    /// it's rather a new contact sharing the same key than the existing one changed its address, otherwise
    /// it would already have our key to encrypt.
    /// This is a regression test for a bug where DC Core wrongly executed AEAP in this case.
```